app/com/linkedin/drelephant/tuning/engine/SparkHBTParamRecommender.java

```diff
@@ -360,4 +430,62 @@ private int getMemoryIncreaseForGC() {
       return 0;
     }
   }
+
+  /**
+   * This method suggest three parameters : spark.dynamicAllocation.minExecutors, spark.dynamicAllocation.maxExecutors
+   *  and spark.executor.instances
+   * For now we are changing these parameters only in case we are changing number of executor core
+   */
+  private void suggestNumExecutorParameters() {
```

Comment:
Should we also check if dynamic allocation is enabled. I mean whats the point of setting it if its not enabled. Saying because its an open source product

Reply:
Done.